### PR TITLE
Added data fetcher and dummy data 

### PR DESCRIPTION
**Pull request by omri08**

### CODE REVIEW

File: pages/index.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
-import Head from 'next/head'
+import Head from "next/head";
+import { DataFetcher, FakeDbStrategy } from "./api";
```
Comment (phil-ociraptor) [inline pages/index.js:2]: This import seems unnecessary. Please remove this file from the PR. Let me show you how to do this in a video

Comment (phil-ociraptor) [reply pages/index.js:2]: https://www.loom.com/share/4dcdaa9e7c084e5db4207118c5ff51a5

The command I want you to run is simply:
```
gco master -- pages/index.js
```

This is a very useful concept to know though!

**Comment by omri08** [comment]
After spending a few hours on it I've just realized that you cant put inside /pages files that are not React component.
So just for now, I've put the data folder outside, maybe we should create a utils folder to put it there? like utils/data

**Comment by omri08** [comment]
Sorry for a lot of commits and pushes, I didn't realize you need to build and run to check before pushing :(

**Pull request merged**